network/tcp.go

```diff
@@ -0,0 +1,97 @@
+// Provides Network Stats (TCP and TCP6)
+package network
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/intelsdi-x/snap-plugin-collector-docker/mounts"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
+)
+
+func TcpStatsFromProc(rootFs string, pid int) (wrapper.TcpStat, error) {
```

Comment:
Added comment to `TcpStatsFromProc` and `Tcp6StatsFromProc`. 

As you notice changing `Tcp -> TCP` and `tcp -> TCP` can be problematic in context of exported and unexported functions, I've just kept this as it is now.